Sources/SourceKitLSP/Swift/ClosureCompletionFormat.swift

```diff
@@ -0,0 +1,64 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+public import SwiftSyntax
+public import SwiftBasicFormat
```

Comment:
We need to support building with a Swift 5.10 compiler as well, which doesn’t support the access level imports.

```suggestion
#if compiler(>=6.0)
public import SwiftSyntax
public import SwiftBasicFormat
#else
import SwiftSyntax
import SwiftBasicFormat
#endif
```

Reply:
Done!